README.md

```diff
@@ -23,6 +23,16 @@ Commits on GitHub can be viewed using web services from the W3C:
  * Rendered page: http://api.csswg.org/bikeshed/?url=https://raw.githubusercontent.com/ome/ngff/master/0.2/index.bs
  * Diff: https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fngff.openmicroscopy.org%2F0.2%2F&doc2=http%3A%2F%2Fapi.csswg.org%2Fbikeshed%2F%3Furl%3Dhttps%3A%2F%2Fraw.githubusercontent.com%2Fome%2Fngff%2Fmaster%2Flatest%2Findex.bs
 
+## New versions
+
+* Make new changes to `latest/index.bs`
+* Update changelog at the bottom of `latest/index.bs`
+* Find references to previous version and _in most cases_, bump to the current version.
+* On completion:
```

Comment:
Is there also an action around changing the status from `Editor's draft` to `Final Community report`?